publication/ver11/societal_impact.md

```diff
@@ -0,0 +1,216 @@
+geolocation information to a WoT Thing Description.  To mitigate privacy
+hazards from both current and future metadata in WoT Thing Descriptions, the 
+WoT Discovery process provides mechanisms to control access to WoT Thing Descriptions.
+
+### 3. [What risks do you see in features of your specification being misused, or used differently from how you intended?](https://w3ctag.github.io/societal-impact-questionnaire/#misuse)
+Many possible misuses and risks are already discussed in the answers to the Security and
+Privacy questionnaire, such as distributed denial of service attacks and
+man-in-the-middle attacks.  See also the individual specifications for 
+descriptions of risks and mitigations.  
+
+In general, in addition to the "normal" risks of access to digital information,
+IoT devices can also provide real-time sensor data (a surveillance risk), and
+IoT actuators could potentially be a safety risk (e.g. an attacker starting a car in
+an enclosed space, filling the garage with carbon monoxide).  Most of these
+already exist without WoT, so what WoT brings is easier access to this functionality.
+It describes what already exists, it does not create it.  Mechanisms provided
+in WoT Discovery, however, can be used to limit access to WoT metadata to those
+who already have access to the associated IoT services.
+
+### 4. [Can users of the Web Platform choose not to use features of your specification?](https://w3ctag.github.io/societal-impact-questionnaire/#opt-out)
+An end user can choose not to install IoT devices.
+They could also use to access IoT services directly without first accessing
+WoT metadata (e.g. using specific documentation for that IoT service)
+although this would in most cases lead to significantly more effort.
+ 
+### 5. [What groups of people are excluded from using features of your specification?](https://w3ctag.github.io/societal-impact-questionnaire/#excluded)
+No specific groups of people are excluded, although language may be a barrier.
+Vocabulary used in the WoT Thing Description is based on English, much like
+HTML tags.  Support is included for including human-readable content in 
+multiple languages but it is up to device manufacturers to include that content.
+Access to the network interfaces of IoT devices would, of course, only be available
+to those with network access and a computer able to generate network requests.
+
+To broaden access, translation services are needed to be able to extend existing
+WoT Thing Models with third-party translations.  This aspect of the WoT Thing Models
+needs further work.  However, the WoT JP CG has been working on translating the
+specifications themselves into Japanese.
+
+### 6. [What effect may features of your specification have on minority groups?](https://w3ctag.github.io/societal-impact-questionnaire/#minority-groups)
+No specific impact, other than the language barrier noted.
+Minority groups may not be able to access metadata in their language.
```

Comment:
I would also mention that this depends on the manufacturer or service provider which languages are supported.

Reply:
note added